config/dev.webpack.config.js

```diff
@@ -5,18 +5,18 @@ const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
     debug: true,
-    ...(process.env.BETA && { deployment: 'beta/apps' }),
+    ...(process.env.BETA && { deployment: 'preview/apps' }),
     ...process.env.PROXY && {
         https: true,
         useProxy: true,
         proxyVerbose: true,
         env: `${process.env.ENVIRONMENT || 'stage'}-${
           process.env.BETA ? 'beta' : 'stable'
         }`, // for accessing prod-beta start your app with ENVIRONMENT=prod and BETA=true
-        appUrl: process.env.BETA ? '/beta/insights/inventory' : '/insights/inventory',
+        appUrl: process.env.BETA ? '/preview/insights/inventory' : '/insights/inventory',
```

Comment:
This is the only place in webpack config that should use preview instead of beta. You can also use both.
```js
        appUrl: process.env.BETA ? ['/beta/insights/inventory' , '/preview/insights/inventory'] : '/insights/inventory',
```